CHANGES.rst

```diff
@@ -225,6 +225,9 @@ Bug Fixes
 
   - Fixed a problem with setting ``bounding_box`` on 1D models. [#5718]
 
+  - Fixed a broadcasting problem with weighted fitting of 2D models
+    with ``LevMarLSQFitter``. [#5737]
```

Comment:
@astrofrog - apparently astrobot was sleeping when this was merged, I'm opening an issue and will try to fix it later.